codecs/mozjpeg/dec/mozjpeg_dec.cpp

```diff
@@ -36,7 +36,7 @@ val decode(std::string image_in) {
   // Process data
   while (cinfo.output_scanline < cinfo.output_height) {
     uint8_t* ptr = &output_buffer[stride * cinfo.output_scanline];
-    jpeg_read_scanlines(&cinfo, &ptr, 1);
+    jpeg_read_scanlines(&cinfo, &ptr, cinfo.output_height);
```

Comment:
If this can be reached multiple times, shouldn't it be containing "how many lines are left" as opposed to total number of lines all the time?

If it's getting too complicated, I'm okay with just reverting to `1` for now and addressing this separately.